etc/docker/test/matrix.yml

```diff
@@ -40,9 +40,9 @@ suites:
       - oracle
       - mysql8
       - postgres9
-      - postgres12
+      - postgres14
```

Comment:
I think this is good, I am wondering though about the `postgres9` one. In principle this is not supported anymore since end of last year. I wonder if we should just test against postgres14 and remove postgres9 entirely.
Raises a bit also the question of our compatibilities we mention in http://rucio.cern.ch/documentation/requirements

Reply:
We only have the old one because psql9 is the only available one in CentOS7. psql10 is default in CentOS8, however psql12 is also available. I think it's fine to just have psql14.

Reply:
As discussed, let's remove everything but postgres14